src/index.html

```diff
@@ -265,6 +265,10 @@
         <title>Star</title>
         <path d="M12 .587l3.668 7.568 8.332 1.151-6.064 5.828 1.48 8.279-7.416-3.967-7.417 3.967 1.481-8.279-6.064-5.828 8.332-1.151z"/>
       </symbol>
+      <symbol id="icon-outline-star" viewBox="0 0 24 24">
+        <title>Star</title>
+        <path d="M12 .587l3.668 7.568 8.332 1.151-6.064 5.828 1.48 8.279-7.416-3.967-7.417 3.967 1.481-8.279-6.064-5.828 8.332-1.151z" stroke="#88a658" stroke-width="2"/>
+      </symbol>
```

Comment:
We're trying to move away from this method of handling SVGs, but since the stars should logically be together and the other star is grandfathered in (for now), I'll let it slide. 😉